rest-api-spec/src/main/resources/rest-api-spec/test/cat.shards/10_basic.yaml

```diff
@@ -86,6 +87,35 @@
 
   - do:
       indices.create:
+        index: sync_id_test
+        body:
+          settings:
+            number_of_shards: 5
+            number_of_replicas: 0
+
+  - do:
+      cluster.health:
+        wait_for_status: green
+  - do:
+      indices.flush_synced:
+        index: sync_id_test
+
+  - is_false: _shards.failed
+
+  - do:
+      cat.shards:
+        index: sync_id_test
+        h: index,sync_id
+  - match:
+      $body: |
+               /^(sync_id_test\s+[A-Za-z0-9_\-]{20}\s+\n){5}$/
```

Comment:
@bleskes That seems like a good idea. Currently the output will be null for such shards. Before I write the test, do you think that is the output that we should go with?